### PR TITLE
[69] Use Travis for continuous integration

### LINKED ISSUE #69: Use Travis for continuous integration

### PR DESCRIPTION
**Pull request by carol-mohemian**
Resolves #69 

I added the `travis.yml` file and set-up the config to run the tests. There are somethings missing like SonarCloud integration and some other "nice to have" stuff, will create issues for these and link them here as well.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: swift
+os: osx
+osx_image: xcode9.2
+xcode_project: JOSESwift.xcodeproj
+xcode_workspace: JOSESwift.xcworkspace
+xcode_scheme: Tests
+
+script:
+  - bundle exec fastlane scan -s JOSESwift
```
Comment (daniel-mohemian) [inline .travis.yml]: does `bundle install` happen automatically?

Comment (carol-mohemian) [reply .travis.yml]: jep 👍

Comment (daniel-mohemian) [reply .travis.yml]: cool how?

Comment (carol-mohemian) [reply .travis.yml]: I guess travis searches for the `Gemfile` 🤔

**Pull request merged**